doc/developer-guides/hld/ivshmem-hld.rst

```diff
@@ -181,3 +181,75 @@ post-launched VMs (VM1 and VM2).
 
    - For VM1 use ``ls -lh /sys/bus/pci/devices/0000:00:06.0/uio``
    - For VM2 use ``ls -lh /sys/bus/pci/devices/0000:00:05.0/uio``
+
+Inter-VM Communication Security hardening (BKMs)
+************************************************
+
+As previously highlighted, ACRN 2.0 provides the capability to create shared
+memory regions between Post-Launch user VMs known as “Inter-VM Communication”. 
+This mechanism is based on ivshmem v1.0 exposing virtual PCI devices for the
+shared regions (in Service VM's memory for this release). This feature adopts a
+community-approved design for shared memory between VMs, following same
+specification for KVM/QEMU (`Link <https://git.qemu.org/?p=qemu.git;a=blob_plain;f=docs/specs/ivshmem-spec.txt;hb=HEAD>`_).
+
+Following the ACRN threat model, the policy definition for allocation and
+assignment of these regions is controlled by the Service VM, which is part of
+ACRN’s Trusted Computing Base (TCB). However, to secure inter-VM communication
+between any userspace applications that harness this channel, applications will
+face more requirements for the confidentiality, integrity, and authenticity of
+shared or transferred data. It is the application development team's
+responsibility to define a threat model and security architecture for the
+application and utilize custom or public libraries accordingly. In this document
+we provide an overview about potential hardening techniques from a userspace
+application's perspective. Consider these techniques when defining the security
+architecture and threat model for your application.
+
+.. note:: This is not a definitive guide on all security technologies or how to
```

Comment:
Thanks for fixing some of the long lines.  Alas, indenting is important for restructuredText so this needs to be written as:
```
.. note:: This is not a definitive guide on all security technologies or how to
   implement security. We provide general pointers not bounded to a specific
   OS or use-case.
```

P.S.  This is the doc build failure being reported by the CI system

Reply:
Thanks for the guidance @dbkinder. Done